x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/script.tsx

```diff
@@ -81,7 +81,7 @@ const fieldsConfig: FieldsConfig = {
   lang: {
     type: FIELD_TYPES.TEXT,
     deserializer: String,
-    serializer: from.undefinedIfValue('painless'),
+    serializer: (v: unknown) => (v === 'painless' || v === '' ? undefined : v),
```

Comment:
[nit] I know this wasn't introduced in this PR - but I noticed that if I set the `lang` to `"painless"` via the ES API, it does retain the value. I think it could be potentially confusing to a user if they add `painless` to the text field in the UI and then it is stripped from the request.